pkgs/os-specific/linux/mba6x_bl/default.nix

```diff
@@ -1,31 +1,24 @@
 { fetchFromGitHub, kernel, stdenv }:
 
-with stdenv.lib;
-
-let pkgName = "mba6x_bl";
-in
-
 stdenv.mkDerivation rec {
-  name = "${pkgName}-${version}";
-  version = "2016-04-22";
+  name = "mba6x_bl";
```

Comment:
The version field here equals the day of commit. Can you keep this scheme?

Reply:
Done!